custom-words.txt

```diff
@@ -1974,6 +1974,7 @@ VMILR
 vmimage
 vmname
 vmotion
+VMOS
```

Comment:
How this is related to Restore Point api changes.

Reply:
when i added this line (containingRestorePointSourceVMOSDisk) to swagger, the automatic spell check flagged VMOS
         "$ref": "#/definitions/RestorePointSourceVMOSDisk",
I have no choice but to add that work to custom-words.txt to get past this spellcheck